ddtrace/llmobs/_evaluators/sampler.py

```diff
@@ -0,0 +1,84 @@
+import json
+from json.decoder import JSONDecodeError
+import os
+from typing import List
+from typing import Optional
+
+from ddtrace import config
+from ddtrace.internal.logger import get_logger
+from ddtrace.sampling_rule import SamplingRule
+
+
+logger = get_logger(__name__)
+
+
+class EvaluatorRunnerSamplingRule(SamplingRule):
+    SAMPLE_RATE_KEY = "sample_rate"
+    EVALUATOR_LABEL_KEY = "evaluator_label"
+    SPAN_NAME_KEY = "span_name"
+
+    def __init__(self, sample_rate: float, evaluator_label: Optional[str] = None, span_name: Optional[str] = None):
```

Comment:
Should `evaluator_label` and `span_name` have a type of `Optional[Any]`, since it could be the default of `NO_RULE`, which is an [`object()`](https://github.com/DataDog/dd-trace-py/blob/b6db0c6a11365602f4a8568c35d7ea3184bf5d80/ddtrace/sampling_rule.py#L27)?

Reply:
That makes sense, updating to `object` (the code quality bot complains about Any)

Reply:
sounds good! saw you made the change, maybe `Optional[Union[str, object]]` would be just a bit clearer, but otherwise idt it really matters (on another note, if we either do a string or default to the object, just `Union[str, object]` is probs fine, but at the end of the day it's just typing 😜 )

Reply:
Ah that's definitely more clear - updated!